mappings/net/minecraft/world/World.mapping

```diff
@@ -47,8 +47,18 @@ CLASS net/minecraft/class_18 net/minecraft/world/World
 	METHOD method_180 setState (Ljava/lang/String;Lnet/minecraft/class_58;)V
 		ARG 1 id
 		ARG 2 state
+	METHOD method_189 findPathToLocation (Lnet/minecraft/class_57;IIIF)Lnet/minecraft/class_61;
+		ARG 1 entity
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+		ARG 5 radius
 	METHOD method_190 getEntityCollisions (Lnet/minecraft/class_57;Lnet/minecraft/class_25;)Ljava/util/List;
 	METHOD method_191 playSound (Lnet/minecraft/class_57;Ljava/lang/String;FF)V
+	METHOD method_192 findPathToEntity (Lnet/minecraft/class_57;Lnet/minecraft/class_57;F)Lnet/minecraft/class_61;
```

Comment:
[findPathTo](https://maven.fabricmc.net/docs/yarn-1.20.6+build.3/net/minecraft/entity/ai/pathing/EntityNavigation.html#findPathTo(net.minecraft.entity.Entity,int)) might be closer match